becquerel/core/spectrum.py

```diff
@@ -132,6 +134,8 @@ def __init__(
             acquisition start
           stop_time (optional): datetime or string representing the acquisition
             end.
+          kwargs (optional): any other named attributes to be stored in the
+            attrs member.
```

Comment:
Yes, I was thinking to drop `infilename` altogether, since each of the parser functions now just set it in the metadata they return. It's only used when printing the spectrum, but there's some code needed to manage it. I'll probably just move it to `Spectrum.attrs` and be done with it.